tasks/winbuildscripts/buildwin.bat

```diff
@@ -1,29 +1,30 @@
 if not exist c:\mnt\ goto nomntdir
-
+@echo on
 @echo c:\mnt found, continuing
 
-mkdir \dev\go\src\github.com\DataDog\datadog-agent
-if not exist \dev\go\src\github.com\DataDog\datadog-agent exit /b 2
-cd \dev\go\src\github.com\DataDog\datadog-agent || exit /b 3
+set BUILD_ROOT=c:\buildroot
+mkdir %BUILD_ROOT%\datadog-agent
+if not exist %BUILD_ROOT%\datadog-agent exit /b 2
+cd %BUILD_ROOT%\datadog-agent || exit /b 3
 xcopy /e/s/h/q c:\mnt\*.* || exit /b 4
 
-call %~p0extract-modcache.bat
+call %BUILD_ROOT%\datadog-agent\tasks\winbuildscripts\extract-modcache.bat
```

Comment:
this has `c:\mnt\modcache.tar.gz` hardcoded. Since `xcopy` is copying everything, might as well do the extraction in the new `BUILD_ROOT`.